default.nix

```diff
@@ -68,6 +68,11 @@ let
 
   rtkernel = callPackage ./kernel { inherit (l4t) l4t-xusb-firmware; realtime = true; };
   rtkernelPackages = (pkgs.linuxPackagesFor rtkernel).extend kernelPackagesOverlay;
+
+  nx-jetson-benchmarks = callPackage ./jetson-benchmarks/default.nix { 
```

Comment:
Could you go ahead and make the other variants?

Reply:
Made other variants and tested them on the hitl rack